client_plugin/drivers/vfile/dockerops/dockerops.go

```diff
@@ -489,7 +502,12 @@ func (d *DockerOps) StopSMBServer(volName string) (int, string, bool) {
 	log.Infof("StopSMBServer for vol %s", volName)
 	serviceID, _, err := d.getServiceIDAndPort(volName)
 	if err != nil {
-		return 0, "", false
+		if err.Error() != noSambaServiceError {
+			return 0, "", false
+		} else {
+			// if the service is already stopped
+			return 0, "", true
```

Comment:
Because the purpose of this function is to stop the service.
When the service is already stopped, we just return true so the event handlers can proceed to next steps.